src/kohlrahbi/ahbtable/ahbpackagetable.py

```diff
@@ -0,0 +1,149 @@
+"""
+class which contains AHB package condition table
+"""
+
+import json
+import re
+from pathlib import Path
+
+import pandas as pd
+from docx.table import Table as DocxTable  # type: ignore[import-untyped]
+from maus.edifact import EdifactFormat
+from pydantic import BaseModel, ConfigDict
+
+from kohlrahbi.logger import logger
+
+# pylint: disable=duplicate-code
```

Comment:
woher kommt das duplicate code?

Reply:
removed in f70dc9761138759ce3b089a604c622114c0605e6

Reply:
I think this commit is not the correct one, but the `disable duplicate code` line seems to be gone 👍